lib/project-data-updater.js

```diff
@@ -44,9 +44,7 @@ function ProjectDataUpdater(config, repository, updateLock, done) {
 
   var that = this;
   this.done = done || function(err) {
-    if (err) {
-      console.log(err);
-    } else {
+    if (err === null) {
```

Comment:
Oh, with one caveat: Still want that initial `err` object; it's a very well-established idiom: http://thenodeway.io/posts/understanding-error-first-callbacks/. But the second argument can be an object similar to what you described.